test/client/index.js

```diff
@@ -54,9 +54,8 @@ module.exports = config => {
     })
   }
 
-  Client.create = function (origin, email, password, options) {
+  Client.create = function (origin, email, password, options={}) {
     var c = new Client(origin)
-    c.options = options || {}
```

Comment:
@hritvi Looks like a mistake here, you updated options, but didn't pass options to `c.options`.